spec.md

```diff
@@ -877,23 +917,25 @@ message GetCapacityRequest {
 }
 
 message GetCapacityResponse {
-  message Result {
-    // The available capacity of the storage that can be used to
-    // provision volumes. If `volume_capabilities` or `parameters` is
-    // specified in the request, the Plugin SHALL take those into
-    // consideration when calculating the available capacity of the
-    // storage. This field is REQUIRED.
-    uint64 available_capacity = 1;
-  }
-
-  // One of the following fields MUST be specified.
-  oneof reply {
-    Result result = 1;
-    Error error = 2;
-  }
+  // The available capacity of the storage that can be used to
+  // provision volumes. If `volume_capabilities` or `parameters` is
+  // specified in the request, the Plugin SHALL take those into
+  // consideration when calculating the available capacity of the
+  // storage. This field is REQUIRED.
+  uint64 available_capacity = 1;
 }
 ```
 
+##### GetCapacity Errors
+
+If the plugin is unable to complete the GetCapacity call successfully, it MUST return a non-ok gRPC code in the gRPC status.
+If the conditions defined below are encountered, the plugin MUST return the specified gRPC error code.
+The CO MUST implement the specified error recovery behavior when it encounters the gRPC error code.
+
+Condition | gRPC Code | Description | Recovery Behavior
+| --- | --- | --- | --- |
+| Unsupported `volume_capability` | 3 INVALID_ARGUMENT | Indicates that one or more of the specified `volume_capability` options (mount flags, volume type, FS type, etc.) are not supported. | Caller MUST fix the `volume_capability` before retrying. |
```

Comment:
This fits into the general invalid/unsupported fields condition? Consider remove it?

Reply:
Will do